types/recharts/index.d.ts

```diff
@@ -645,13 +646,13 @@ export interface TooltipProps {
 	itemStyle?: any;
 	wrapperStyle?: any;
 	labelStyle?: any;
-	cursor?: boolean | any | React.ReactElement<any>;
+	cursor?: boolean | any | React.ReactElement<any> | React.StatelessComponent<any>;
 	viewBox: ViewBox;
 	active?: boolean;
 	coordinate?: Coordinate;
 	payload?: TooltipPayload[];
 	label?: string | number;
-	content?: React.ReactElement<any> | RechartsFunction;
+	content?: React.ReactElement<any> | React.StatelessComponent<any> | RechartsFunction;
```

Comment:
The same as the question before. The docs states: `ReactElement | Function`

Reply:
Same as before.